### ISSUE TITLE
Introduce 18+ / NSFW tag

### ISSUE THREAD

**Issue by KevinPayravi** [KevinPayravi/indie-wiki-buddy#830]
A few wikis that have been submitted to IWB are adult-oriented. For these wikis, we may want to introduce an `18+` (or similar) tag. Any wikis with this tag would be labelled as 18+ on Indie Wiki Buddy's settings page and the [website's listings](https://getindie.wiki/listings/), just so anyone going through those lists isn't taken by surprise if they click into it.

Note, games that have a few mature references wouldn't get this tag - this is mostly for games that primarily feature adult content (namely the erotic dating and roleplaying games).

**Comment by bayazidbh** [comment]
Yes, that would be nice. I would like for NSFW wikis to be redirected as well, but I agree that on the listing it needs to be warned about.

Two other wikis that I think would belong in this category would be the [Monster Girl Quest](https://mgq.miraheze.org/wiki/MonsterGirlQuest_Wiki) and [Monster Girl Encyclopedia](https://monstergirlencyclopedia.miraheze.org/wiki/Monster_Girl_Encyclopedia) wikis, though I'm not as in-depth in those fandom as I am with AliceSoft. Should I still make an Issue thread for it pointing to this Issue nonetheless?
